randovania/data/json_data/prime3.txt

```diff
@@ -1167,9 +1332,11 @@ Asset id: 7492844587871276588
 > Dock to Hidden Court Hall; Heals? False
   * Normal Door to Hidden Court Hall/Dock to Fuel Gel Pool
   > Pickup (Missile Expansion - Ledge)
-      Ship Grapple and Ice Missile
+      Ice Missile and After Fuel Gel Pool Golem Head
   > Pickup (Missile Expansion - Bottom)
       Suit Type ≥ 4
+  > Event - Pickup Golem Head
+      Ship Grapple
```

Comment:
Ledge Missile doesn't require the head to be moved. Only Ice is required. Moving the head grants access to the lower item, so the the requirement is on the wrong connection.